### DISCUSSION TITLE
how to style data-radix-popper-content-wrapper

### DISCUSSION THREAD

**Discussion by ogheneovo12** [radix-ui/primitives#2310] [Help]
Need help in styling The Popover Content Wrapper, but been unable to figure it out

**Comment by joaom00** [comment]
That doesn't need to be styled. What do you want to achieve?

**Reply by ogheneovo12** [reply]
initially i had wanted to set a higher z-index context for the content, had to use the portal container props for that, as the className for the portal is not being passed to any element.

Then recently i wanted to set the width of the popover content to a specific custom size, without tampering with the related global css vars.

Using a width unit of `vw` and max-width on the content, helped fixed this.

**Reply by joaom00** [reply]
> initially i had wanted to set a higher z-index context for the content, had to use the portal container props for that, as the className for the portal is not being passed to any element.

You can set `z-index` on the `Content` and it will be set on the popper content wrapper too
